bazel/repositories.bzl

```diff
@@ -1,7 +1,7 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-ENVOY_COMMIT = "d3a90127f8c408651e8ab44e961408ca1fd972b4"  # July 07, 2021
-ENVOY_SHA = "4aed1171848e1cf4f5309d918b384b9d34aa1e7f577398caeb1d4f32c8004ee4"
+ENVOY_COMMIT = "5ab509b54bf6916235b5f2b65800b9d756032129"  # July 07, 2021
```

Comment:
We should also update the date in the comment.

Reply:
Done.